app/views/users/sessions/new.html.erb

```diff
@@ -1,5 +1,10 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+  <% content_for :before_content, govuk_back_link(
```

Comment:
This could be on one line

Reply:
Yes, I probably copied it from somewhere where it was split. I'll change it